src/Drupal/Driver/Cores/Drupal6.php

```diff
@@ -506,4 +506,18 @@ public function clearStaticCaches() {
     // Drupal 6 doesn't have a way of clearing all static caches.
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function entityCreate($entity_type, $entity) {
+    // Drupal 6 doesn't have an entity API.
```

Comment:
This should instead probably be added to `BaseDriver` where it throws an exception instead of silently failing.